platform/ui/src/components/ViewportActionBar/ViewportActionBar.jsx

```diff
@@ -14,7 +14,7 @@ const ViewportActionBar = ({
   studyData,
   showNavArrows,
   showPatientInfo,
-  onSeriesChange,
+  onNavigationClick,
```

Comment:
I agree. This really should be an empty container where you can optionally render all these bits as children